src/main/java/bio/terra/app/controller/JobsApiController.java

```diff
@@ -94,32 +93,30 @@ private AuthenticatedUserRequest getAuthenticatedInfo() {
   // -- jobs --
   @Override
   public ResponseEntity<List<JobModel>> enumerateJobs(
-      @RequestParam(value = "offset", required = false, defaultValue = "0") Integer offset,
-      @RequestParam(value = "limit", required = false, defaultValue = "10") Integer limit,
-      @RequestParam(value = "direction", required = false, defaultValue = "desc")
-          SqlSortDirection direction,
-      @RequestParam(value = "flightClass", required = false, defaultValue = "")
-          String flightClass) {
-    validiateOffsetAndLimit(offset, limit);
+      Integer offset,
```

Comment:
Do you know why the defaults for other parameters are honored without the annotations while the one for `direction` is not?

Reply:
Oh, because those are directly set in the openapi spec.  E.g.


<img width="786" alt="image" src="https://user-images.githubusercontent.com/5633787/189161016-8afc618e-599e-4541-97fe-8e09549fae5f.png">


Note: offset and limit have defaults, sort direction can't (because open api 3.0 doesn't support mixing in parameters with refs), and className doesn't have one

Reply:
Thanks!